REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.3
 MathProgBase 0.3.8
 ECOS 0.4.1
+FactCheck 0.2.5
```

Comment:
Done, thanks! I'm going to keep this PR open for another few days to see if anyone has other feedback, and merge it then.